aws/requests.yaml

```diff
@@ -21,6 +21,8 @@ releases:
   requests:
   - name: coredns
     version: ">= 1.14.2"
+  - name: observability-bundle
+    version: ">= 0.2.0"
```

Comment:
Does `>=` mean future releases will always retrieve the latest version of o11y-bundle when released?

Reply:
I think that's thé Idea yes but let's make sure we update that 😅

Reply:
means if we have 0.2.0 CI will not break and we might NOT bump any further. If you need a newer version you might want to bump in the requests

Reply:
Can I have an approval still?